src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -21,15 +23,42 @@ public class TestJobs2dPatterns {
 	 * @param application Application context.
 	 */
 	private static void setupPresetTests(Application application) {
-		SelectTestFigureOptionListener selectTestFigureOptionListener1 = new SelectTestFigureOptionListener(
-				DriverFeature.getDriverManager(), 1);
 
-		SelectTestFigureOptionListener selectTestFigureOptionListener2 = new SelectTestFigureOptionListener(
-				DriverFeature.getDriverManager(), 2);
+        SelectTestFigureOptionListener selectTestFigureOptionListener1 = null;
+        SelectTestFigureOptionListener selectTestFigureOptionListener2 = null;
 
-		application.addTest("Figure Joe 1", selectTestFigureOptionListener1);
-		application.addTest("Figure Joe 2", selectTestFigureOptionListener2);
-	}
+		SelectTestFigureOptionListener selectTestRectangleListener = null;
+		SelectTestFigureOptionListener selectTestTriangleListener = null;
+		SelectTestFigureOptionListener selectTestCircleListener = null;
+
+        try {
+
+            selectTestFigureOptionListener1 = new SelectTestFigureOptionListener(
+                    DriverFeature.getDriverManager(), Script.FIGURE_SCRIPT_1);
+
+            selectTestFigureOptionListener2 = new SelectTestFigureOptionListener(
+                    DriverFeature.getDriverManager(), Script.FIGURE_SCRIPT_2);
+
+			selectTestTriangleListener = new SelectTestFigureOptionListener(
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.